.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: Close Stale Pull Requests
+
+on:
+  schedule:
+    #Runs at 00:00 every day
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+     - uses: actions/checkout@master
+     - name: stale
+       uses: gatsbyjs/stale@master
+       with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DAYS_BEFORE_STALE: 30
+          DAYS_BEFORE_CLOSE: 7
+          STALE_PR_LABEL: 'Stale'
+          STALE_PR_MESSAGE: 'Hi, were are going to mark this PR as stale because it has been inactive for the past 30 days. If no further activity occurs within the following 7 days, it will be automatically closed so that others can take up the issue.If you are still working on this PR, please make a follow-up commit within 7 days and leave a comment to remove the stale label. Do let us know if you are stuck so that we can help you!'
```

Comment:
It's easier to format using to use block scalar. Use [yaml-multiline.info](yaml-multiline.info) to visual how it works.



```
STALE_PR_MESSAGE: | 

Hi, 

We're are going to mark this PR as stale because it has been inactive for the past 30 days.
 
If no further activity occurs within the following 7 days, it will be automatically closed so that others can take up the issue. 
If you are still working on this PR, please make a follow-up commit within 7 days and leave a comment to remove the stale label.

Do let us know if you are stuck so that we can help you!
```